CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/documentation/articles/app-insights-release-notes-dotnet/).
 
-## Version 2.9.0-beta1
+## Version 2.9.0-beta2 (there won't be a beta1 for 2.9.0)
```

Comment:
remove everything after the beta2.
i don't know what that extra comment will do to the automation scripts.

you can safely add those comments on a new line.